.github/workflows/build.yml

```diff
@@ -47,9 +46,13 @@ jobs:
         java-version: 11
     - name: Setup Apache Qpid Broker j
       run: |
-        wget https://downloads.apache.org/qpid/broker-j/9.1.0/apache-qpid-broker-j-9.1.0-src.tar.gz
-        tar -xvzf apache-qpid-broker-j-9.1.0-src.tar.gz
-        cd apache-qpid-broker-j-9.1.0-src/broker/bin
+        wget https://downloads.apache.org/qpid/broker-j/9.1.0/binaries/apache-qpid-broker-j-9.1.0-bin.tar.gz
+        tar -xvzf apache-qpid-broker-j-9.1.0-bin.tar.gz
+        cd qpid-broker/9.1.0/bin/
+        nohup ./qpid-server start &
```

Comment:
There is additional start command at line 56. Is it needed ?

Reply:
Yes, That is to start the Qpid server again after the config file update.

Reply:
do we need to start before config file update as well ?